docs/content/en/docs/getting-started/install-docker.md

```diff
@@ -23,15 +23,39 @@ The easiest way to start experimenting with Tetragon is to run it via Docker
 using the released container images.
 
 ```shell
-docker run --name tetragon-container --rm --pull always \
+docker run -d --name tetragon --rm --pull always \
```

Comment:
btw why did we change that? I think running interactively was fine to get the logs and everything.

Reply:
This made things a bit more similar between Docker and Kubernetes.